src/main/java/com/github/alexcojocaru/mojo/elasticsearch/StartElasticsearchNodeMojo.java

```diff
@@ -105,7 +110,7 @@ public void execute() throws MojoExecutionException
 
     protected ElasticsearchNode startNode(Settings settings) throws MojoExecutionException
     {
-        if (getNode() != null)
+        if (reuseElasticNode && getNode() != null)
```

Comment:
My bad for not reading the description you added to the merge request; I understand now why you need this parameter.

If reuseElasticNode is false, then the node property is reset and you loose the reference to the previously started ES instance.